release/RELEASE.md

```diff
@@ -122,10 +141,13 @@
    "All manual tests have passed".
 
 1. Finally, announce the release on the relevant internal Slack channels.
-   An announcement on the external Slack channel linking to our blog should appear
-   automatically.
 
-1. Documentation is published automatically once the release is public on Github,
-   however it takes up to an hour to complete.
+1. *[STABLE]* Coordinate with product (& marketing) for the relevant public
+   announcements (public Slack, Twitter, etc.).
+   should appear automatically.
+
+1. *[STABLE]* Documentation is published automatically once the release is
```

Comment:
Not really, no. We'd just need to figure out how/where to publish them. My current thinking (but this can easily be changed after this PR) is that prereleases should stay marked as prereleases on GitHub, and that currently makes them invisible to the docs cron.